OpenLive-Web/src/pages/meeting.jsx

```diff
@@ -166,12 +175,11 @@ const MeetingPage = () => {
               })
               .then(() => {
                 localClient.publish()
-                mutationCtx.setScreen(true)
+                // mutationCtx.setScreen(true)
```

Comment:
Hey I'm not sure about who's the maintainer, but imho this won't change as this is common when a client joins and would certainly affect other functionalities.